### PR TITLE
Button slot

### PR DESCRIPTION
**Pull request by joshua-larks**

### CODE REVIEW

File: src/components/DemoSection.vue [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+  get escapedHtml() {
+    const codeRegEx = new RegExp(
+      `title="${
+        this.title
+      }" :code="demoCode">\\s*<template #components>(.*?)<\\/template>`,
+      "gsm"
+    );
+
+    const codeMatch = codeRegEx.exec(this.code) as string[];
```
Comment (hoodsy) [inline src/components/DemoSection.vue]: @joshua-larks have never seen anything like this before - how does `as string[]` work?? (just out of curiosity)

same with `let indentedLines: string[] = [];`

File: src/components/FormGroupV.vue [hunk 2]
```diff
@@ -1,8 +1,12 @@
 <template>
   <div class="s-form-group-v">
     <!-- title -->
-    <div class="s-form-group-v__title" :style="titleLayoutStyle">
-      <label>{{ title }}</label>
+    <div
+      class="s-form-group-v__title"
+      :class="{ 's-form-group-v__title--hidden': !title && !tooltip }"
+      :style="titleLayoutStyle"
```
Comment (hoodsy) [inline src/components/FormGroupV.vue]: @joshua-larks consider putting style first for pretty formatting :smile: 
```
      :style="titleLayoutStyle"
      class="s-form-group-v__title"
      :class="{ 's-form-group-v__title--hidden': !title && !tooltip }"
```

maybe u need `style` last for overrides tho.

File: src/components/DemoSection.vue (hunk 1 above)
Comment (joshua-larks) [reply src/components/DemoSection.vue]: Oh its typescript. Just setting the type to an array of strings

File: src/components/FormGroupV.vue (hunk 2 above)
Comment (joshua-larks) [reply src/components/FormGroupV.vue]: Can switch

**Pull request merged**